### ISSUE TITLE
no_implicit_braces false positive with `extends a.b`

### ISSUE THREAD

**Issue by lydell** [clutchski/coffeelint#549]
```
$ cat coffeelint.json
{
  "no_implicit_braces": {"level": "error"}
}
$ cat tst.coffee
class A extends B.C
  constructor: ->
$ bin/coffeelint tst.coffee
  ✗ tst.coffee
     ✗ #2: Implicit braces are forbidden.

✗ Lint! » 1 error and 0 warnings in 1 file
```

Workaround:

```
{C} = B
class A extends C
  constructor: ->
```

Use case: https://github.com/akhodakivskiy/VimFx/commit/0f280108c6a7bd84027c663abb34c4a03fefb091

Tested with latest master (commit d8df418ce7cecb4ef5645c806f5de7b231aefbbf).
